test_plans/intel_device_info.asciidoc

```diff
@@ -0,0 +1,142 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for Intel's Extensions for Device Information
+
+This is a test plan for the APIs described in
+https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md[sycl_ext_intel_device_info]
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_INTEL_DEVICE_INFO` so they can be skipped
+if feature is not supported.
+
+== Initial setup
+
+[source, c++]
+----
+auto devices = device::get_devices();
+
+std::for_each(devices.cbegin(), devices.cend(), [](const auto& dev) {
+  /* tests */
+});
+----
+
+Get available devices by calling `get_devices()` and run the code in the sections below for each device.
+Code should consist checking if device has aspect for this `info` and checking type.
```

Comment:
This paragraph contradicts the "Device coverage" section, where you say that the tests are only run on the device that is selected on the CTS command line.  Our normal policy is to run CTS tests only that selected device.  Why should this test be different?

Reply:
Forgot about that, removed.